### PR TITLE
archive exhibition page

### PR DESCRIPTION
**Pull request by ostafinskim**

### CODE REVIEW

File: resources/views/components/exhibition-cta.blade.php [hunk 1]
```diff
@@ -1,9 +1,21 @@
+@php
+    /*
+        Carbon::createFromFormat('Y-m-d', $exhibition["exhibition_end_date"])->endOfDay()->isPast()
+        Is checking if the end date of an exhibition is in the past.
+        If the end date is today, it will return false until the end of the day (23:59:59).
```
Comment (sean-dunwoody) [inline resources/views/components/exhibition-cta.blade.php:5]: good commenting 👍

File: routes/breadcrumbs.php [hunk 2]
```diff
@@ -333,7 +333,7 @@
 Breadcrumbs::for('archive', function (BreadcrumbTrail $trail): void
 {
     $trail->parent('home');
-    $trail->push('Visit Us',route('visit'));
+    $trail->push('Visit Us', '/plan-your-visit');
```
Comment (sean-dunwoody) [inline routes/breadcrumbs.php:336]: Why was this changed? Surely using the `route` method is more reliable than hard coding the URL like this?

**Review by sean-dunwoody — APPROVED** [review]
Looks good to me @ostafinskim . I've just added one query about part of the code.

File: routes/breadcrumbs.php (hunk 2 above)
Comment (ostafinskim) [reply routes/breadcrumbs.php]: Hi @sean-dunwoody for some reason when I worked on this yesterday my local one didn't like `route('visit') when I tried preview all archive.

I tested this again and it works fine, so I removed this.

Thanks for spotting this

**Review by JamieC24 — APPROVED** [review]
